fuel-indexer-macros/src/indexer.rs

```diff
@@ -448,6 +487,16 @@ fn process_fn_items(
                                 decoder.decode_logdata(rb, data);
 
                             }
+                            Receipt::ScriptResult { result, gas_used } => {
+                                #contract_conditional
+                                let data = ScriptResult{ result: u64::from(result), gas_used };
```

Comment:
I believe you can remove the `u64::from`, as the decimal string will be automatically coerced to `u64`